### PR TITLE
Add css grid framework to master

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

**Comment by discodavey** [comment]
@nlisgo Can you look at this and then I can merge into patterns-php and journal for pr check.

File: source/_patterns/04-pages/article--research.mustache [hunk 1]
```diff
@@ -37,6 +37,9 @@
             {{#articleSections}}
               {{> molecules-article-section }}
             {{/articleSections}}
+            {{# annotationCount}}
```
Comment (nlisgo) [inline source/_patterns/04-pages/article--research.mustache:40]: Why is this change necessary?

File: assets/sass/patterns/molecules/view-selector.scss [hunk 2]
```diff
@@ -151,15 +151,16 @@
...
-    @include constrain-width(210, "max");
-    width: 9.625rem;
+    grid-column: 1/3;
+    grid-row: 1;
   }
 
   .view-selector--fixed {
+    @include padding(48, "top");
     max-height: 100vh;
+    max-width: 164px;
```
Comment (nlisgo) [inline assets/sass/patterns/molecules/view-selector.scss:161]: Should we use `rem` here also?

File: assets/sass/_mixins.scss [hunk 3]
```diff
@@ -273,17 +273,17 @@
   margin: auto;
 
   @if $padding {
-    padding-left: 7%;
-    padding-right: 7%;
+    padding-left: #{$site-spacing}px;
```
Comment (nlisgo) [inline assets/sass/_mixins.scss:276]: Should we be using `rem` as we do in the media query?

Comment (discodavey) [reply assets/sass/_mixins.scss:276]: I am following how the rest of the spacing is in the variables stylesheet

**Comment by discodavey** [comment]
Closing as no longer needed. Explaination is on main ticket 7321
